ppr-ui/src/views/newMhrRegistration/MhrReviewConfirm.vue

```diff
@@ -215,14 +216,10 @@ export default defineComponent({
           scrollToInvalid(MhrSectVal.LOCATION_VALID, 'mhr-home-location')
           break
         case RouteNames.MHR_REVIEW_CONFIRM:
+          let stepsValidation = getSteps.value.filter((step : StepIF) => step.valid)
```

Comment:
Filtering only valid steps? Do we need extra conditions here?
Since it's an MHR Registration flow as well, we just need to be cautious to not break it.

Reply:
Sorry that was a typo. It should be map step .valid

Reply:
Ok, but interestingly enough that all tests were still passing. I know we don't have 100% coverage, so that's why we need to be careful when changing something in the existing code.

Reply:
Agreed. Ill merge this PR to move this ticket along, and ill work on separate PR to add some tests for scrolling functionality.